### ISSUE TITLE
Feature request: Auto-sort option for structure field items

### ISSUE THREAD

**Issue by texnixe** [getkirby/panel#652]
See this post https://forum.getkirby.com/t/auto-sort-structure-field-entries-in-panel/2226. It should be possible to define the field(s) to sort by.

``` text
fields:
  publications:
    label: Publications
    type: structure
    autosort:
     - published
     - title
    fields:
      published:
        label: Publication date
        type: date
      title:
        label: Title
        type: text
      description:
        label: Description
        type: textarea
```

**Comment by creichel** [comment]
:+1:

**Comment by NECKRHINOS** [comment]
I'd like to see this as well, plus #666

**Comment by distantnative** [comment]
Similar to #722 – might consider multiple fields for the PR.

**Comment by lukasbestle** [comment]
This feature is now available in v2.5.0.
